stetho/build.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
     compile 'com.google.code.findbugs:jsr305:2.0.1'
 
     compile 'com.android.support:appcompat-v7:23.0.1' // optional
+    compile 'net.zetetic:android-database-sqlcipher:3.5.7@aar'
```

Comment:
You appear to have a number of unrelated changes here that should be removed.  Also an aside, you don't need this SQLCipher thing directly in Stetho as you can just install your own `DatabaseDriver2` implementation now (see `SqliteDatabaseDriver` and work from there as needed).